clio-server/src/main/scala/org/broadinstitute/clio/server/service/SchemaService.scala

```diff
@@ -1,189 +0,0 @@
-package org.broadinstitute.clio.server.service
-
-import java.time.OffsetDateTime
-
-import io.circe._
-import org.broadinstitute.clio.transfer.model._
-import shapeless._
-import shapeless.labelled._
-
-import scala.concurrent.{ExecutionContext, Future}
-import scala.reflect.runtime.universe.{Type, TypeTag, typeOf, typeTag}
-
-/**
-  * Generate JSON schema from case classes such as the QueryOutputs.
-  *
-  * JsonSchema uses the same shapeless pattern as the
-  * [[org.broadinstitute.clio.util.generic.FieldMapper]]
-  */
-object SchemaService {
-
-  /**
-    * a JSON schema for A suitable for Circe
-    *
-    * @tparam A is a case class such as a QueryOutput
-    */
-  trait JsonSchema[A] {
```

Comment:
`JsonSchema` got moved to transfer-model so the ITs could access its instances, and after that there wasn't much point to keeping the `SchemaService` around.